src/app/ui/color_wheel.cpp

```diff
@@ -84,6 +84,22 @@ app::Color ColorWheel::getColorInClientPos(const gfx::Point& pos)
   int v = (pos.y - (m_wheelBounds.y+m_wheelBounds.h/2));
   double d = std::sqrt(u*u + v*v);
 
+  if (m_colorModel == ColorModel::NORMAL_MAP) {
+    float x = float(u) / float(m_wheelBounds.w / 2);
+    float y = -float(v) / float(m_wheelBounds.h / 2);
+    float z = std::sqrt(1 - x*x - y*y);
```

Comment:
I think here we should use the distance `d` (distance with the center of the wheel), and the angle `a` (which is already calculated below as `double a = std::atan2(-v, u)`) instead of `x` and `y` (as it's now it looks like a box instead of a wheel)